ui/src/FindBar.tsx

```diff
@@ -1,23 +1,53 @@
 import React from "react";
+import { convertToAnnotationId } from './selectors/annotation'
 import { EventBus } from "./types/pdfjs-viewer";
+import { ScholarReaderContext } from "./state";
+import { BoundingBox } from "./types/api";
+
+interface FindBarProps {
+  mappingToBounds: Map<String, BoundingBox>;
+  matches: string[];
+  selectedSymbol: string;
+}
 
 interface FindBarState {
   currentMatch: number | null;
   matchCount: number | null;
+  mode: string,
```

Comment:
Thought: this might help make the available types for the mode explicit:

```suggestion
  mode: "hidden" | "text-search" | "symbol-search",
```